scripts/hooks/prevent-illegal-characters.sh

```diff
@@ -23,11 +23,12 @@ function check_conflict_marker() {
     done
 }
 
-function check_addEventListener_wrong_usage() { 
+function check_addEventListener_wrong_usage() {
     # git search query returning file names that used window.addEventListener illegally
     illegal_files=$(git diff  --cached --diff-filter=ACMR \
                     -G 'window.addEventListener' --name-only  -- core \
                     ':(exclude)scripts/hooks/prevent-illegal-characters.sh' \
+                    ':(exclude)core/examples/luigi-core-material-ui/src/serviceWorker.js' \
```

Comment:
Did you use a script to generate the app? I guess we can remove `serviceWorker.js` file to keep the example clean and simple if possible.

Reply:
done